core/templates/domain/utilities/language-util.service.spec.ts

```diff
@@ -83,31 +83,31 @@ describe('Language util service', function() {
 
   it('should get the correct language direction given a content language code',
     () => {
-      expect(lus.getContentLanguageDirection('en')).toEqual('ltr');
-      expect(lus.getContentLanguageDirection('ar')).toEqual('rtl');
-      expect(lus.getContentLanguageDirection('bg')).toEqual('ltr');
+      expect(lus.getLanguageDirection('en')).toEqual('ltr');
+      expect(lus.getLanguageDirection('ar')).toEqual('rtl');
+      expect(lus.getLanguageDirection('bg')).toEqual('ltr');
     }
   );
 
-  // eslint-disable-next-line max-len
-  it('should get "auto" language direction when given an invalid content language code',
+  it('should get throw an error when given an invalid content language code',
```

Comment:
I think there's a typo? 'get' should be removed. Ditto below.

Reply:
Fixed.